benchmark/httpapi.py

```diff
@@ -114,10 +137,11 @@ def post(self, request):
         request.setHeader(b'content-type', b'application/json')
         try:
             json = loads(request.content.read())
+            timestamp_parser.parse(json['timestamp'])
```

Comment:
:grey_question: This is sort of a design-y comment, and I'm not sure how robust the benchmark-server needs to be. So I'll leave optional for now. If implemented it should possibly even be a new PR.

It occurs to me that https://github.com/ClusterHQ/benchmark-server/pull/3/files#diff-d7efc1c0aaf2abe86cade065973bd61cR52 sort of assumes that this check has been run. It would be nice if that was codified, although that might not be worth the refactor that would be involved. Something like:

```
class StoredValue(PClass):
   timestamp = field(type=timestamp)
   json = field(type=dict)

....
  # In the interface
  def store(result):
     """
     ...
     :param StoredValue result: The result to store.
     ...
     """
...
  # And then here:
  try:
     stored_value = StoredValue(timestamp=timestamp_parse.parse(json['timestamp']),
                                                 json=json)
  except  ....
   ...
   self.backend.store(stored_value)
```

Then get_timestamp would just return value.timestamp, and would not be able to have create KeyErrors or ValueErrors, even if it gets out of sync with the code here.